mongodb.md

```diff
@@ -0,0 +1,85 @@
+# MongoDB
+
+TODO add summary and links to sections
```

Comment:
Could we also add a link to our [quick start](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/quick-start/), perhaps towards the end of the doc?